pom.xml

```
@@ -133,7 +133,7 @@
     <tagNameFormat>formatter-m2e-configurator-@{project.version}</tagNameFormat>
 
     <takari.release.gpg.skip>true</takari.release.gpg.skip>
-    <tychoVersion>4.0.13</tychoVersion>
+    <tychoVersion>5.0.0</tychoVersion>
```

Updating tycho is more complicated than a version bump.

```suggestion
    <tychoVersion>4.0.13</tychoVersion>
```